### PR TITLE
get trace file's detail via /trace/:name/log_detail

### PR DESCRIPTION
**Pull request by zhongwencool**
When users view the trace logs in the cluster, 
they generally look directly at the files on the nodes that have been updated recently, 
so we need a new API to return the trace file's size and modification times.

image

### CODE REVIEW

File: CHANGES-5.0.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Add `cert_common_name` and `cert_subject` placeholder support for authz_http and authz_mongo.[#8973](https://github.com/emqx/emqx/pull/8973)
 * Use milliseconds internally in emqx_delayed to store the publish time, improving precision.[#9060](https://github.com/emqx/emqx/pull/9060)
 * More rigorous checking of flapping to improve stability of the system. [#9136](https://github.com/emqx/emqx/pull/9136)
+* Add /trace/:name/log_detail HTTP API to return trace file's size and mtime.
```
Comment (ieQu1) [inline CHANGES-5.0.md:8]: Missing link to the PR

File: apps/emqx_management/src/proto/emqx_mgmt_trace_proto_v2.erl [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+trace_file_detail(Nodes, File) ->
+    rpc:multicall(Nodes, emqx_trace, trace_file_detail, [File], 25000).
+
+-spec read_trace_file(node(), binary(), non_neg_integer(), non_neg_integer()) ->
+    {ok, binary()}
+    | {error, _}
+    | {eof, non_neg_integer()}
+    | {badrpc, _}.
+read_trace_file(Node, Name, Position, Limit) ->
+    rpc:call(Node, emqx_mgmt_api_trace, read_trace_file, [Name, Position, Limit]).
```
Comment (thalesmg) [inline apps/emqx_management/src/proto/emqx_mgmt_trace_proto_v2.erl:66]: Just checking: should this also have a timeout like the other RPC calls?

File: apps/emqx_management/src/emqx_mgmt_api_trace.erl [hunk 3]
```diff
@@ -136,6 +144,19 @@ schema("/trace/:name/download") ->
             }
         }
     };
+schema("/trace/:name/log_detail") ->
+    #{
+        'operationId' => log_file_detail,
+        get => #{
+            description => "get trace log file's detail message, such as size, last update time",
```
Comment (thalesmg) [inline apps/emqx_management/src/emqx_mgmt_api_trace.erl:151]: Maybe:

```suggestion
            description => "get trace log file's metadata, such as size, last update time",
```

?

File: apps/emqx_management/src/proto/emqx_mgmt_trace_proto_v2.erl (hunk 2 above)
Comment (zhongwencool) [reply apps/emqx_management/src/proto/emqx_mgmt_trace_proto_v2.erl:66]: great!

**Pull request merged**